config.yaml

```diff
@@ -46,3 +46,15 @@ options:
       depends on this full backup also expires.
       This option is EXPRERIMENTAL.
       Allowed values are: from 1 to 9999999.
+  access-key:
+    type: string
+    description: |
+      Access Key (account) for connecting to the object storage.
+      This config option takes precedence over values passed via the sync-s3-credentials action.
+      This value must be unset for the action to have effect.
+  secret-key:
```

Comment:
If we are handling secrets, then we should use [user secrets](https://juju.is/docs/juju/secret#heading--user) here. [One example](https://github.com/canonical/object-storage-integrators/blob/66814cf21e2a8dfcd14a7efcb5f35471e6f6aa87/config.yaml#L20)